features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/MessagesView.kt

```diff
@@ -463,22 +467,31 @@ private fun MessagesViewTopBar(
             BackButton(onClick = onBackClick)
         },
         title = {
-            val roundedCornerShape = RoundedCornerShape(8.dp)
-            val titleModifier = Modifier
-                .clip(roundedCornerShape)
-                .clickable { onRoomDetailsClick() }
-            if (roomName != null && roomAvatar != null) {
-                RoomAvatarAndNameRow(
-                    roomName = roomName,
-                    roomAvatar = roomAvatar,
-                    heroes = heroes,
-                    modifier = titleModifier
-                )
-            } else {
-                IconTitlePlaceholdersRowMolecule(
-                    iconSize = AvatarSize.TimelineRoom.dp,
-                    modifier = titleModifier
-                )
+            Row(
+                horizontalArrangement = Arrangement.spacedBy(4.dp),
+                verticalAlignment = Alignment.CenterVertically,
+            ) {
+                val roundedCornerShape = RoundedCornerShape(8.dp)
+                val titleModifier = Modifier
+                    .clip(roundedCornerShape)
+                    .clickable { onRoomDetailsClick() }
+                if (roomName != null && roomAvatar != null) {
+                    RoomAvatarAndNameRow(
+                        roomName = roomName,
+                        roomAvatar = roomAvatar,
+                        heroes = heroes,
+                        modifier = titleModifier
+                    )
+                } else {
+                    IconTitlePlaceholdersRowMolecule(
+                        iconSize = AvatarSize.TimelineRoom.dp,
+                        modifier = titleModifier
+                    )
+                }
+
+                if (isDmUserVerified) {
+                    Icon(imageVector = CompoundIcons.Verified(), tint = ElementTheme.colors.iconSuccessPrimary, contentDescription = null)
+                }
```

Comment:
This icon will not be visible if the room name is too long:

<img width="299" alt="image" src="https://github.com/user-attachments/assets/598e8709-aa42-45c2-b80e-f5a553806956" />

Or its size will be reduced.

<img width="301" alt="image" src="https://github.com/user-attachments/assets/ca08c483-fd61-4515-b014-247453db7e57" />